vysledek/style.css

```diff
@@ -0,0 +1,327 @@
+*, ::after, ::before { box-sizing: border-box; }
+
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #f5f5f5;
+    font-family: 'Merriweather', serif;
+    font-size: 16px;
+    line-height: 1.6;
+}
+
+.leading {
+    display: flex;
+    flex-basis: 100%;
+    flex-direction: column;
+    font-family: 'Raleway', sans-serif;
+    font-weight: 600;
+    justify-content: space-around;
+    min-width: 440px;
+    background-color: white;
+
+}
+
+/*                  HEADER                              */
+header {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    background-image: url("/images/island-hlavicka.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    color: white;
+    text-align: center;
+    font-weight: 700;
+    padding-left: 25px;
+    padding-right: 25px;
+    padding-bottom: 25px;
+    min-width: 440px;
+    font-family: 'Raleway', sans-serif;
+}
+
+
+header h1 {
+    font-size: 3.5rem;
+}
+
+.signature {
+    color: lightgray;
+    position: relative;
+}
+
+.signature::before {
+    position:absolute;
+    content: '';
+    top:-20px;
+    height: 3px;
+    margin: 0 auto;
+    right: 0;
+    left: 0;
+    width: 17%;
+    background: lightgray;
+}
+/*--------------                                    */
+
+
+
+/*                  NAVIGACE                        */
+nav a {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: lightgray;
+}
+
+ul {
```

Comment:
Tady bych použila nějaký konkrétnější selektor (třeba ul uvnitř nav), takhle to nastavujeme pro všechny ul obecně.